scripts/mocha.sh

```diff
@@ -7,5 +7,6 @@ set -ex
 mocha \
   -c \
   --require test.config.js \
```

Comment:
I think this line will have to be removed?

Reply:
Nope, it's still needed for any shared stuff.

Reply:
Oh so there's not a total separation (because I believe jest still uses that file). If they _need_ to be separate it might be worth moving shared dependencies into a third location and creating a `moca.config.js` and a `jest.config.js` file that import `config.common` (or something).

Reply:
Both jest and mocha use that file. There may need to be a separation, but I'd rather that be handled in another PR. This has been... trying.